Assets/Labrys/FeatureEditor/AggSectionField.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections.Generic;
+
+namespace Labrys.FeatureEditor
+{
+	public class AggSectionField : IField
+	{
+		public List<IField> Fields { get; set; } = new List<IField>();
+
+		public string Name
+		{
+			get
+			{
+				string aggregate = "";
+				foreach (IField f in Fields)
+				{
+					if (aggregate == "")
+						aggregate = f.Name;
```

Comment:
```
why
    are
        we
            so
                indented
                    ?
```

Reply:
wait these are tabs

Reply:
streus no

Reply:
Going to convert everything to spaces after this PR.